<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFwRulesDlg</class>
 <widget class="QDialog" name="EditFwRulesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VM Firewall</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QRadioButton" name="policyAllowRadioButton">
     <property name="text">
      <string>Allow network access except...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="policyDenyRadioButton">
     <property name="text">
      <string>Deny network access except...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="rulesTreeView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dnsCheckBox">
         <property name="text">
          <string>Allow DNS queries</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="newRuleButton">
         <property name="text">
          <string>&amp;New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editRuleButton">
         <property name="text">
          <string>&amp;Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRuleButton">
         <property name="text">
          <string>&amp;Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>policyAllowRadioButton</tabstop>
  <tabstop>policyDenyRadioButton</tabstop>
  <tabstop>dnsCheckBox</tabstop>
  <tabstop>rulesTreeView</tabstop>
  <tabstop>newRuleButton</tabstop>
  <tabstop>editRuleButton</tabstop>
  <tabstop>deleteRuleButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFwRulesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditFwRulesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
